src/app/home/home.component.html

```diff
@@ -1,3 +1,4 @@
 <sw-banner swScrollSpy id="banner"></sw-banner>
 <sw-countdown></sw-countdown>
 <sw-about swScrollSpy id="about"></sw-about>
+<sw-events></sw-events>
```

Comment:
Add the scrollspy directive for your component, and add the link in the navbar too.